### ISSUE TITLE
Fig asks for the accessibility permission again (and files permission) after update  

### ISSUE THREAD

**Issue by Ronserruya** [withfig/fig#799]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have typed 'git ' and included a screenshot of the Debugger (click the Fig icon in the macOS menu bar)

### Description:
After #706 was solved(thanks!) I updated fig (clicked on fig > check for update > install (or whatever the rightmost one is)).  
After the update, Fig asked me to enable the accessibility permission again, even though it was already enabled, so I needed to tick it off and on again.  
Also, Fig requested access to folders again (open terminal, cd Desktop, and get an access dialog to allow fig access to the Desktop folder), even though I gave permissions in the previous version

<img width="1185" alt="Screen Shot 2022-01-19 at 14 51 53" src="https://user-images.githubusercontent.com/21235163/150135191-148f727b-0475-47aa-bc98-7cdb0aeeb88b.png">

<img width="1075" alt="Screen Shot 2022-01-19 at 14 39 45" src="https://user-images.githubusercontent.com/21235163/150135043-09d55ed8-c72e-439e-8f3e-d8be10a6b368.png">

### Environment

```bash
Fig Version: Version 1.0.54 (B359) [ABC]
UserShell: /usr/local/bin/bash
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /usr/local/bin/fig
Accessibility: true
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path:
iTerm Integration: installed!
Hyper Integration: application is not present.
VSCode Integration: application is not present.
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installed via Brew: true
Installation Script: true
PseudoTerminal Path: /Users/ron/.sdkman/candidates/java/current/bin:/usr/local/opt/gnu-tar/libexec/gnubin:/usr/local/opt/grep/libexec/gnubin:/Users/ron/.nvm/versions/node/v12.13.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/share/dotnet:~/.dotnet/tools:/Users/ron/.fig/bin:/Users/ron/Library/Python/3.8/bin:/Library/Frameworks/Python.framework/Versions/3.8/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: bash (95059) - /dev/ttys002
Current working directory: /Users/ron
Current window identifier: 229614/% (com.googlecode.iterm2)
Path: /Users/ron/.sdkman/candidates/java/current/bin:/usr/local/opt/gnu-tar/libexec/gnubin:/usr/local/opt/grep/libexec/gnubin:/Users/ron/.nvm/versions/node/v12.13.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/share/dotnet:~/.dotnet/tools:/Users/ron/.fig/bin:/Users/ron/Library/Python/3.8/bin:/Library/Frameworks/Python.framework/Versions/3.8/bin
Fig environment variables:
  - TERM_SESSION_ID=w0t0p0:0906A9DA-40E3-4A4D-839F-4EA879064E2B
  - FIG_ENV_VAR=1
  - FIG_INTEGRATION_VERSION=5
  - FIG_TERM=1
  - FIG_TERM_VERSION=3
  - FIG_CHECKED_PROMPTS=1
MacOS Version: 10.15.7 19H2
Hardware:
  - Model Name: MacBook Pro
  - Model Identifier: MacBookPro15,1
  - Chip:
  - Cores: 6
  - Memory: 16 GB
```

**Comment by mschrage** [comment]
Hey @Ronserruya! This is a known issue for this update since we switched from my personal Apple Developer account to a company account.

Shouldn't happen going forward. But I'll leave this issue open for now.

**Comment by brendanfalk** [comment]
As an FYI for any users seeing this, you may have to toggle accessibility off then on again to fix (even if it's checked)
